packages/cli/src/api/catalog.ts

```diff
@@ -202,12 +202,16 @@ export class Catalog {
     namespace?: CompiledCatalogNamespace
   ) {
     let ext: string
-    if (namespace === "es") {
-      ext = "mjs"
-    } else if (namespace === "ts") {
-      ext = "ts"
-    } else {
-      ext = "js"
+    switch (namespace) {
+      case "es":
+        ext = "mjs"
+        break
+      case "ts":
```

Comment:
Did you forget to port the "ts" case?

```js
ext = "ts"
break
```

I'm surprised that the test doesn't catch it!

Reply:
Ah nevermind I just understood what you did!